src/net/sourceforge/kolmafia/request/UseSkillRequest.java

```diff
@@ -536,13 +536,17 @@ public long getMaximumCast() {
         return 1;
 
       case SkillPool.CALCULATE_THE_UNIVERSE:
-        if (KoLCharacter.getAdventuresLeft() == 0) {
-          return 0;
+        {
+          if (KoLCharacter.getAdventuresLeft() == 0) {
+            return 0;
+          }
+          int skillLevel = Preferences.getInteger("skillLevel144");
+          if (KoLCharacter.isHardcore() || KoLCharacter.inRonin()) {
```

Comment:
elsewhere we can `!KoLCharacter.canInteract()` for a check like this

Reply:
OK.